### PR TITLE
Configure RailsAdmin to use webpacker

### PR DESCRIPTION
**Pull request by raccube**

### CODE REVIEW

**Comment by pixeldesu** [comment]
Is there a specific reason that makes this beneficial if Webpacker is deprecated and I'm already in the process of thinking of removing it?

**Comment by raccube** [comment]
After running `assets:precompile` for production locally, I was unable to get any assets to build until I did this.

**Comment by raccube** [comment]
RailsAdmin won't support cssbundling/jsbundling until 3.1 (which is in beta)

File: app/javascript/packs/rails_admin.js [hunk 1]
```diff
@@ -0,0 +1,2 @@
+import "rails_admin/src/rails_admin/base";
+import "../stylesheets/rails_admin.scss";
```
Comment (pixeldesu) [inline app/javascript/packs/rails_admin.js]: I'd preferably have this entrypoint as a `.ts` file, if that works. Contents don't need to be modified for that!

**Review by pixeldesu — CHANGES_REQUESTED** [review]

File: app/javascript/packs/rails_admin.js (hunk 1 above)
Comment (raccube) [reply app/javascript/packs/rails_admin.js]: 555f47d

**Review by pixeldesu — APPROVED** [review]
👍 LGTM

**Pull request merged**